lms/static/sass/base/_animations.scss

```diff
@@ -258,6 +258,18 @@
   }
 }
 
+@mixin fade-transition {
+  -webkit-transition: all 0.2s ease-out;
+    -moz-transition: all 0.2s ease-out;
```

Comment:
Also, be sure you want to transition all CSS property changes here. This will find any diffs in properties on elements you're calling this from.